.ci/wercker.sh

```diff
@@ -276,6 +276,50 @@ no-exception-alot-of-projects)
   rm -rf contribution
   ;;
 
+no-error-imports-guava)
+  PROJECTS=checks-import-order/projects-to-test-imports-guava.properties
+  CONFIG=checks-import-order/checks-imports-error-guava.xml
+  REPORT=reports/guava/site/index.html
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo CS_version: ${CS_POM_VERSION}
+  checkout_from https://github.com/checkstyle/contribution.git
+  cd .ci-temp/contribution/checkstyle-tester
+  groovy ./launch.groovy --listOfProjects $PROJECTS --config $CONFIG \
+      --checkstyleVersion ${CS_POM_VERSION}
+  SUMMARY=(`sed -n "s/^<td>\(.*\)<.td>.*/\1/p" < $REPORT`)
```

Comment:
please do `grep -R -A 5 "&#160;Warning</td>" . | cat` , and do exit 1 if content is not empty. If will be good to print content before exit.

Reply:
done